src/components/Articles.jsx

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { Grid } from '@mui/material';
+
+import ArticleCard from './ArticleCard';
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+  console.log('🚀 ~ Articles ~ articles:', articles);
+  const [queries, setQueries] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get('https://news-site-backend-project.onrender.com/api/articles')
+      .then(({ data }) => {
```

Comment:
You can extract this axios get request to a utils file and import it in as a function. This will clean up code in your component